### PR TITLE
write feature pull request template

### PR DESCRIPTION
**Pull request by buxx**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+*Please indicate here a summary of pull request content or if related issue exist, please indicate issue reference here*
+
+## Checkpoints
+
+The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
+If one or more of these checkpoints are out of scope, please write below why they are.
+
+- [ ] Code is intelligible enought
```
Comment (lebouquetin) [inline .github/PULL_REQUEST_TEMPLATE/default.md]: `Code is intelligible enought` => `Code is clear enough`

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] If there is FIXME in code, these is an associated issue and issue reference is indicate in FIXME
```
Comment (lebouquetin) [inline .github/PULL_REQUEST_TEMPLATE/default.md]: `If there is FIXME in code, these is an associated issue and issue reference is indicate in FIXME` => `If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME`

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] If there is TODO or NOTE, or HACK in code, date and developper initials are present
```
Comment (lebouquetin) [inline .github/PULL_REQUEST_TEMPLATE/default.md]: `If there is TODO or NOTE, or HACK in code, date and developper initials are present` => `If there are TODO, NOTE or HACK in code, date and developer initials are present`

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Automated tests have been write (covering feature, or non regression)
```
Comment (lebouquetin) [inline .github/PULL_REQUEST_TEMPLATE/default.md]: `Automated tests have been write (covering feature, or non regression)` => `Automated tests have been write (covering feature or non regression), or *at least* an issue has been created for test implementation`

Comment (Skylsmoi) [reply .github/PULL_REQUEST_TEMPLATE/default.md]: "have been written"

**Pull request merged**